src/geode/mesh/builder/solid_mesh_builder.cpp

```diff
@@ -348,10 +394,7 @@ namespace geode
             for( auto&& edge_vertices :
                 solid_mesh_->polyhedron_edges_vertices( added_polyhedron ) )
             {
-                if( edge_vertices[0] < edge_vertices[1] )
-                {
-                    edges.find_or_create_edge( std::move( edge_vertices ) );
-                }
```

Comment:
Vertices are always in this order since creation of `polyhedron_edges_vertices`-like functions

Reply:
not sure. In any case, this is not the responsability of the solid builder to filter this.